packages/perseus-editor/src/article-editor.tsx

```diff
@@ -151,91 +151,103 @@ export default class ArticleEditor extends React.Component<Props, State> {
         } as const;
 
         const sections = this._sections();
+        const editingDisabled = this.props.apiOptions?.editingDisabled ?? false;
 
         return (
             <div className="perseus-editor-table">
```

Comment:
Not something you need to address in this PR, but the number if indents here is kinda gross. Maybe at a later time, it could be worth breaking the inner section out into a different file for readability.

Reply:
As I've worked with the editing components, I've thought there's an opportunity to "componentize" some of these structures that are here.. the whole stack of views that are made up by these `perseus-editor-row` and `perseus-editor-(left|right)-cell` feel like they're components waiting to be extracted. :)